doc/source/utils_findsat_mrt.py

```diff
@@ -0,0 +1,9 @@
+.. _utils_findsat_mrt:
+
+**************************
+MRT-based Satellite Trails Detection Utilities
+**************************
```

Comment:
Section underline should match the number of characters.

Reply:
This file was actually out of date and should have been removed from the repository. Now done.